base.js

```diff
@@ -151,6 +152,26 @@ const baseConfig = {
         // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md
         'no-shadow': OFF,
         '@typescript-eslint/no-shadow': ERROR,
+
+        // These two rules deal with autofixing type imports/exports
+        // https://typescript-eslint.io/rules/consistent-type-imports
+        '@typescript-eslint/consistent-type-imports': [
+          ERROR,
+          { fixStyle: 'inline-type-imports' },
```

Comment:
Should probably bump the [peerDep](https://github.com/seek-oss/eslint-config-seek/blob/master/package.json#L49) to 4.5 now too.